yarn.lock

```diff
@@ -7887,7 +7905,31 @@ amplify-cli-core@2.4.3:
     typescript-json-schema "^0.51.0"
     which "^2.0.2"
 
-amplify-codegen@2.26.22, amplify-codegen@^2.23.1:
+amplify-codegen@2.26.23:
+  version "2.26.23"
+  resolved "https://registry.npmjs.org/amplify-codegen/-/amplify-codegen-2.26.23.tgz#1ffb2d6c585c887c701fbc1e7813c05848106c92"
+  integrity sha512-MQN1XSCQ8BQ6NVyTTaM31VDyQ0JUEBvJfLe8+LmqVVJ7LR0gaLKTfR5QmSjjhRyEnUjE0ukMciPwGBTygIgkWg==
+  dependencies:
+    "@aws-amplify/appsync-modelgen-plugin" "1.29.13"
+    "@aws-amplify/graphql-docs-generator" "2.4.2"
+    "@aws-amplify/graphql-types-generator" "2.8.6"
+    "@graphql-codegen/core" "1.8.3"
+    amplify-codegen-appsync-model-plugin "^1.22.3"
+    amplify-graphql-docs-generator "^2.2.1"
+    amplify-graphql-types-generator "^2.7.0"
+    chalk "^3.0.0"
+    fs-extra "^8.1.0"
+    glob-all "^3.1.0"
+    glob-parent "^5.1.1"
+    graphql "^14.5.8"
+    graphql-config "^2.2.1"
+    inquirer "^7.3.3"
+    js-yaml "^4.0.0"
+    ora "^4.0.3"
+    semver "^7.3.5"
+    slash "^3.0.0"
+
+amplify-codegen@^2.23.1:
   version "2.26.22"
   resolved "https://registry.yarnpkg.com/amplify-codegen/-/amplify-codegen-2.26.22.tgz#fecbbc49ed3f8a68de15022888a3d2bc47f6c237"
```

Comment:
Any idea why we still have 2.26.22 being resolved?

Reply:
@alharris-at I've opened https://github.com/aws-amplify/amplify-cli/pull/9490 to address the issue. A PR was merged today that was pulling in an outdated version of `amplify-provider-awscloudformation`, which was still depending on the older version of codegen.